### PR TITLE
Pull Request for #5: Cluster standard errors at the county-level

### LINKED ISSUE #5: Cluster standard errors at the county-level
Opening this issue to cluster standard errors at the county-level as per step 9 of [Practice Task 2: Sherlock Extension](https://github.com/gentzkow/lab-manual-archive/wiki/Practice-Task#2-sherlock-extension).

### PR DESCRIPTION
**Pull request by Xingtong-Jiang**
**Linked Issue:** Resolves #5.

Opening this pull request to merge the development branch in issue #5 with the main branch. 

Changes made in the development branch:
- Revised the code in [analyze_data.py](https://github.com/Xingtong-Jiang/template/blob/4f8e79f73976dc8313e13a3d61cec546e7328f59/analysis/code/analyze_data.py) to cluster standard errors at the county-level. 
- Added two tables to the paper that include the updated standard errors.

Deliverables:
- Updated version of [paper](https://github.com/Xingtong-Jiang/template/blob/4f8e79f73976dc8313e13a3d61cec546e7328f59/paper_slides/output/paper.pdf) (page 5). See the draftable comparison [here](https://draftable.com/compare/AdZyWiAvVsZQ).

Thank you for reviewing the changes! @ShiqiYang2022

### CODE REVIEW

File: analysis/log/make.log [hunk 1]
```diff
@@ -1,5 +1,5 @@
 --------------------------------------------------------------------------------
-Makelog started: 2024-08-15 01:43:19
+Makelog started: 2024-08-22 18:05:45
 Working directory: /Users/xtj/Documents/github/template/analysis
```
Comment (ShiqiYang2022) [inline analysis/log/make.log:3]: From the path, it seems like the results are not generated from Sherlock? Ideally we should push results from Sherlock to Github, see the example [here](https://github.com/ShiqiYang2022/template/commit/1738f543942570145d50eece26750fa586ec2812).

**Review by ShiqiYang2022 — CHANGES_REQUESTED** [review]
@Xingtong-Jiang Thanks for all your great work here!

Everything looks great except one place where I left one comment. After addressing that I am willing to sign off.

File: analysis/log/make.log (hunk 1 above)
Comment (Xingtong-Jiang) [reply analysis/log/make.log]: Thanks for pointing that out! I fixed this in ca61a31. The results are now being generated from Sherlock.

File: texput.log
Comment (ShiqiYang2022) [inline texput.log:1]: @Xingtong-Jiang Why we have this log file pushed to the repo? Shall we should drop this file as it's not necessary?

**Review by ShiqiYang2022 — CHANGES_REQUESTED** [review]

File: texput.log
Comment (Xingtong-Jiang) [reply texput.log:1]: @ShiqiYang2022 Thanks! I’ll be more careful when checking file changes next time. I removed `textput.log` in f465267.

**Review by ShiqiYang2022 — APPROVED** [review]
Go time!

**Pull request merged**